### PR TITLE
Included Table organism within full width text

### PR DESCRIPTION
**Pull request by kave**
https://github.com/cfpb/cfgov-refresh/issues/1419

@KimberlyMunoz 

screen shot 2016-02-10 at 9 01 01 am

### CODE REVIEW

File: cfgov/scripts/archive_events.py [hunk 1]
```diff
@@ -24,10 +24,11 @@ def run():
             for child in events.get_children():
                 event = child.specific
                 if isinstance(event, EventPage):
-                    if event.end_dt < timezone.now():
-                        if event.can_move_to(archived_events):
-                            event.move(archived_events, pos='last-child')
-                            print event.title + ' Event .....archived'
+                    if event.end_dt:
```
Comment (jimmynotjim) [inline cfgov/scripts/archive_events.py]: Is this supposed to be in this PR?

**Comment by KimberlyMunoz** [comment]
I just noticed the tables are created as insets. Is there a way for them to be full-width and still be inside the full-width-text organism?

File: cfgov/scripts/archive_events.py (hunk 1 above)
Comment (kave) [inline cfgov/scripts/archive_events.py]: lolz its a small bug fix, i noticed while working on this branch so i wanted to just get it in.

**Comment by kave** [comment]
@KimberlyMunoz done! 
screen shot 2016-02-11 at 5 04 29 pm

**Comment by KimberlyMunoz** [comment]
:+1:

**Pull request merged**